text-to-speech/General_TTS_inference_API_notebook_version2.ipynb

```diff
@@ -172,8 +168,8 @@
     {
       "cell_type": "code",
       "source": [
-        "#Replace comment section below with your auth token\n",
-        "!ngrok config add-authtoken #place your access token here"
+        "token=userdata.get('ngrok_token')\n",
```

Comment:
can we capitalize the token name and also the key so it's consistent with other notebooks. `"NGROK_TOKEN = userdata.get('NGROK_TOKEN')\n",`